graphql/resolvers.go

```diff
@@ -3154,49 +3154,24 @@ func (r *taskResolver) IsPerfPluginEnabled(ctx context.Context, obj *restModel.A
 	if !evergreen.IsFinishedTaskStatus(utility.FromStringPtr(obj.Status)) {
 		return false, nil
 	}
-	flags, err := evergreen.GetServiceFlags()
+	projectMatches := model.IsPerfEnabledForProject(*obj.ProjectId)
```

Comment:
ProjectMatches doesn't really fit anymore; could just call IsPerfEnabled directly in the if statement